cobigen-cli/cli/src/main/resources/pom.xml

```diff
@@ -64,17 +64,17 @@
     <dependency>
       <groupId>net.sf.m-m-m</groupId>
       <artifactId>mmm-code-java-parser</artifactId>
-      <version>1.0.0-beta6</version>
+      <version>1.0.0-beta7</version>
     </dependency>
     <dependency>
       <groupId>net.sf.m-m-m</groupId>
       <artifactId>mmm-code-base</artifactId>
-      <version>1.0.0-beta6</version>
+      <version>1.0.0-beta7</version>
     </dependency>
     <dependency>
       <groupId>net.sf.m-m-m</groupId>
       <artifactId>mmm-code-java-maven</artifactId>
-      <version>1.0.0-beta6</version>
+      <version>1.0.0-beta7</version>
     </dependency>
     <dependency>
       <groupId>net.sf.m-m-m</groupId>
```

Comment:
why do we manage mmm dependencies here as well? sounds really strange

Reply:
I've removed them now.